.cargo/config.toml

```diff
@@ -5,4 +5,7 @@ rustflags = ["-C", "link-args=-Wl,-undefined,dynamic_lookup"]
 rustflags = ["-C", "link-args=-Wl,-undefined,dynamic_lookup"]
 
 [target.aarch64-apple-darwin]
-rustflags = ["-C", "link-args=-Wl,-undefined,dynamic_lookup"]
\ No newline at end of file
+rustflags = ["-C", "link-args=-Wl,-undefined,dynamic_lookup"]
+
+[target.x86_64-pc-windows-msvc]
+linker = "rust-lld"
```

Comment:
It's still error:
![image](https://github.com/tursodatabase/turso-client-php/assets/82234190/fe877819-5481-47d8-8339-e2cadda7d9e7)

I am also add:
```yml
- name: Install Visual Studio Build Tools
        if: matrix.os == 'windows-2019'
        shell: pwsh
        run: |
          Invoke-RestMethod -Uri https://aka.ms/vs/17/release/vs_buildtools.exe -OutFile vs_buildtools.exe
          Start-Process -FilePath vs_buildtools.exe -ArgumentList "--add", "Microsoft.VisualStudio.Component.VC.Tools.x86.x64", "--add", "Microsoft.VisualStudio.Component.Windows10SDK.19041", "--norestart", "--passive", "--wait" -Wait -PassThru
          Remove-Item vs_buildtools.exe
```